MovieChecker/MovieChecker/BoxOfficeCell.h

```diff
@@ -0,0 +1,17 @@
+//
+//  BoxOfficeCell.h
+//  MovieChecker
+//
+//  Created by Joseph Goldberg on 1/22/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface BoxOfficeCell : UITableViewCell
+
+@property (nonatomic, weak) IBOutlet UILabel *nameLabel;
+@property (nonatomic, weak) IBOutlet UILabel *salesLabel;
+@property (nonatomic, weak) IBOutlet UIImageView *thumbnailImageView;
```

Comment:
This is fine for a first iteration; generally, it's preferable to have the cell subclass expose a method like `updateWithSomeModel:(SomeModel *)someModel`, from which the actual UI elements are set, and keep these `IBOutlet`s private.